server/cli/init/base-file-sources/hooks.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+/* eslint-disable no-var, prefer-arrow-callback */
+var path = require('path');
+var cucumber = require('cucumber');
+var constants = require('../../../constants')
+var outputDir = path.join(__dirname, '../', constants.REPORT_DIR)
+var fs = require('fs');
+ /* eslint-disable new-cap */
+var jsonFormatter = cucumber.Listener.JsonFormatter();
+
+
+var tractorReportHook = function () {
```

Comment:
I believe exporting the json formatter listener(cucumber.Listener.JsonFormatter) will do the job.